docs/sources/next/javascript-api/k6-experimental/fs/open.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'open'
+excerpt: 'open opens a file and returns a promise resolving to a File instance.'
+weight: 20
+---
+
+# open
+
+The `open` function opens a file and returns a promise that resolves to a [File](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/fs/file) instance. This function loads the file into shared memory space accessible by all Virtual Users (VUs). This approach significantly reduces the memory footprint, enabling the handling of large files without overwhelming memory resources.
```

Comment:
```suggestion
The `open` function opens a file and returns a promise that resolves to a [File](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/fs/file) instance. This function loads the file into shared memory space accessible by all Virtual Users (VUs). This approach significantly ~reduces the memory footprint~, enabling the handling of large files without overwhelming memory resources.
```
Of course, it compares with the old `open` function but maybe the reader is not aware of it.

Reply:
I compared it to root-level/native (I don't know how to call it) `open` function somewhere indeed, I could probably also do it here  👍🏻

Reply:
@oleiade I see you applied the suggestion, was not my intent to add it. It was mostly for signaling the place that I was referencing. Sorry, for not being clear.

Doing directly a concrete suggestion:
```
s/memory footprint/memory footprint from the same usage done using [open](link to builtin)/
```